### ISSUE TITLE
False Negative with jdk

### ISSUE THREAD

**Issue by xenoterracide** [uber/NullAway#434]
when using `System.console()` I did not get an error that my return might be null

```java
System.console().readline();
```
did not result in a compile error. My guess this is because NullAway assumes not null by default. This is dangerous. I suspect it would be better to assume any output is be nullable (unless a primitive) and any input is not null.

**Comment by msridhar** [comment]
Hi @xenoterracide this is a deliberate design decision to make NullAway usable.  If we pessimistically assumed that any library routine could return null and might not accept a null parameter, users would be overwhelmed with warnings.  See Section 5 of [this paper](https://manu.sridharan.net/files/FSE19NullAway.pdf) for details.

We use library models to opt out of this optimistic handling of libraries.  Our default library models are here:

https://github.com/uber/NullAway/blob/d7e96e471df5c3420301ba782a3ac59b3ae1430a/nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java#L217

I am in favor of adding a default model of `System.console()` as returning `@Nullable`.  If you'd like to open a PR that'd be great!
